src/pages/Community.tsx

```diff
@@ -292,12 +318,24 @@ const Community: React.FC = () => {
             ))}
           </Box>
         </TabPanel>
+        {/* SECTION - Ratings */}
         <TabPanel value={activeTab} index={1}>
           <Box sx={{ width: "100%", alignItems: "stretch" }}>
             <CommunityRatingForm
               id={id}
               addCommunityRating={addCommunityRating}
             />
+            <ToggleButtonGroup
```

Comment:
Folgendes Problem: Wenn man einen review macht, wird dieser review per default unten angehänket, obwohl per default nach most recent gefiltert wird --> das rating object sollte als erstes in der liste angehänked werden und nicht als letztes. (siehe post dort machen wir es auch schon so).